packages/stellar-axelar-std/src/tests/storage.rs

```diff
@@ -7,58 +7,62 @@ use crate::contractstorage;
 #[contract]
 pub struct Contract;
 
-#[contractstorage]
-pub enum DataKey {
-    #[instance]
-    #[value(u32)]
-    Counter,
-
-    #[persistent]
-    #[value(String)]
-    Message { sender: Address },
-
-    #[temporary]
-    #[value(Address)]
-    LastCaller { timestamp: u64 },
-
-    #[persistent]
-    #[value(bool)]
-    Flag { key: String, owner: Address },
+mod storage {
+    use super::*;
+
+    #[contractstorage]
```

Comment:
To ensure that the storage schema doesn't accidentally break, make the macro auto-create a unit test that does goldie::assert over the below enum definition / token stream?